pkg/operator/staticpod/installerpod/copy.go

```diff
@@ -16,54 +16,52 @@ import (
 // It will retry on API server connection errors except not found error which is fatal for non-optional secrets.
 // In case the secret is optional and we fail to get it, no error is returned and the secret returning is nil.
 func (o *InstallOptions) getSecretWithRetry(ctx context.Context, secretNamePrefix string, isOptional bool) (*v1.Secret, error) {
-	var secret *v1.Secret
-	retryErr := wait.PollImmediateUntil(200*time.Millisecond, func() (done bool, err error) {
-		secret, err = o.KubeClient.CoreV1().Secrets(o.Namespace).Get(o.nameFor(secretNamePrefix), metav1.GetOptions{})
+	var resultSecret *v1.Secret = nil
```

Comment:
that's better

Reply:
i did whatever I can to make this error proof, so if we still getting empty secret I blame client-go